core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -705,5 +706,5 @@ protected Class defineClassFromData(File container, byte[] classData, String cla
     }
 
     public static boolean useAntClassLoader = SystemProperties.getBoolean(ClassicPluginStrategy.class.getName()+".useAntClassLoader");
-    public static boolean DISABLE_TRANSFORMER = SystemProperties.getBoolean(ClassicPluginStrategy.class.getName()+".noBytecodeTransformer");
+    public static final boolean DISABLE_TRANSFORMER = SystemProperties.getBoolean(ClassicPluginStrategy.class.getName()+".noBytecodeTransformer");
```

Comment:
This is not recommended. We want properties to be controllable from System Groovy scripts if needed

Reply:
Oops, I tried to drop all of those (as noted above)... This wasn't intended.